### ISSUE TITLE
Why is short-range potential LR_rc != Rws, the image radius?

### ISSUE THREAD

**Issue by Paul-St-Young** [QMCPACK/qmcpack#699]
`Uniform3DGridLayout::SetLRCutoffs` duplicates the functionality of `LatticeAnalyzer<T,3>::calcSimulationCellRadius` and sets the short-range cutoff for the Coulomb potential to the simulation cell radius (Rsc). This is different from the default short-range cutoff radius of Jastrow and back flow functions set in `BsplineJastrowBuilder` and `BackflowBuilder`, respectively. The Bspline functor cutoff radius defaults to the image radius a.k.a. radius of the Cartesian space Wigner-Seitz cell (Rws).

For a non-orthorhombic cell, Rws>Rsc. My understanding is that Rws is the optimal cutoff radius for short-range functions. Therefore it bothers me to see the potential cutoff set to Rsc. Since the potential cutoff rc has no effect on the Ewald sum (so long as rc*kc is constant), is there any reason not to set rc=Rws? This would make potential and wavefunction consistent and help avoid confusion between Rsc and Rws.

**Comment by prckent** [comment]
Agreed. I think you have found an historical inconsistency. Rc=Rws would be more consistent.

**Comment by prckent** [comment]
Continues to be worth someone fixing this inconsistency.
